src/main/java/edu/harvard/iq/dataverse/datasetutility/AddReplaceFileHelper.java

```diff
@@ -1410,8 +1416,17 @@ private boolean step_060_addFilesViaIngestService(){
             this.addErrorSevere(getBundleErr("final_file_list_empty"));                
             return false;
         }
-
-        ingestService.addFiles(workingVersion, finalFileList);
+        
+        try{
+        for(DataFile dataFile : finalFileList){
+            commandEngine.submit(new CreateDataFileCommand(dataFile, workingVersion, dvRequestService.getDataverseRequest() ));
```

Comment:
As of fa57c53 I'm getting a `NullPointerException` when I run `test_001_AddFileGood` in `FilesIT.java`.

Reply:
Request Service was null. Fixed by using the already available dvRequest included in the constructor.